README.md

```diff
@@ -68,3 +68,11 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+# Screenshots
```

Comment:
It is not shown. Have you tested?

<img width="989" alt="image" src="https://user-images.githubusercontent.com/16056824/188745257-4328d51f-0061-4f7d-bb6d-27666d43e01d.png">